src/helpers/cssSnapshots.ts

```diff
@@ -23,10 +23,15 @@ const flattenClassNames = (
 export const enum FileTypes {
   scss = 'scss',
   less = 'less',
+  css = 'css',
 }
 
 export const getFileType = (fileName: string) =>
-  fileName.endsWith('less') ? FileTypes.less : FileTypes.scss;
+  fileName.endsWith('.css')
```

Comment:
Rather than a nested ternary, we could use `switch` or `if` statements. If it's CSS or LESS, return those types, otherwise assume SASS.